the-blue-alliance-ios/View Controllers/Events/Event/EventViewController.swift

```diff
@@ -121,10 +123,8 @@ extension EventViewController: EventRankingsViewControllerDelegate {
 }
 
 extension EventViewController: MatchesViewControllerDelegate {
-
     func matchSelected(_ match: Match) {
-        let matchViewController = MatchViewController(match: match, myTBA: myTBA, persistentContainer: persistentContainer, tbaKit: tbaKit, userDefaults: userDefaults)
-        self.navigationController?.pushViewController(matchViewController, animated: true)
+        self.matchViewController = MatchViewController(match: match, statusService: statusService, urlOpener: urlOpener, myTBA: myTBA, persistentContainer: persistentContainer, tbaKit: tbaKit, userDefaults: userDefaults)
```

Comment:
We shouldn't have to hold on to this reference anymore, right? We should be able to remove this and the private property above